PetProject/Screens/GamesList/ViewModel+VC/GamesListViewController.swift

```diff
@@ -7,13 +7,40 @@
 
 import UIKit
 import SwiftUI
+import Combine
 
-class GamesListViewController: UIHostingController<GamesListView> {
-    init(gameListVM: GamesListViewModel) {
+protocol GamesListDelegate { }
+
+final class GamesListViewController: UIHostingController<GamesListView>, UISearchBarDelegate {
+    
+    private let gameListVM: GamesListViewModel
+    typealias GamesListViewControllerDelegate = (GamesListDelegate & AlertShowable)
+    private let delegate: GamesListViewControllerDelegate
+    private var cancellables: Set<AnyCancellable> = []
+    
+    init(gameListVM: GamesListViewModel, delegate: GamesListViewControllerDelegate) {
+        self.gameListVM = gameListVM
+        self.delegate = delegate
         let gameListView = GamesListView(gameListVM: gameListVM)
         super.init(rootView: gameListView)
     }
-    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        gameListVM.$errorMessage.sink { [weak self] message in
+            if let errorMessage = message {
+                let alert = UIAlertController(title: "Error".localized, message: errorMessage, preferredStyle: .alert)
+                let okAction = UIAlertAction(title: "Ok".localized, style: .default) { _ in
+                    DispatchQueue.main.async {
```

Comment:
Instead, subscribe to the publisher in the main queue (gameListVM.$errorMessage...)

Reply:
Done